errtracker/errtracker.go

```diff
@@ -56,11 +57,31 @@ var (
 	mockedHostSnapd = ""
 	mockedCoreSnapd = ""
 
-	snapConfineProfile = "/etc/apparmor.d/usr.lib.snapd.snap-confine"
+	snapConfineProfile  = "/etc/apparmor.d/usr.lib.snapd.snap-confine"
+	whoopsiePreferences = "/etc/whoopsie"
 
 	timeNow = time.Now
 )
 
+func whoopsieEnabled() bool {
+	cfg := goconfigparser.New()
+	err := cfg.ReadFile(whoopsiePreferences)
+	if os.IsNotExist(err) {
+		return true
```

Comment:
Two things, have a variable default := true and return it in the appropriate places below.

I also wonder if we should reverse the default. Whoopsie is ubuntu specific after all (right?)